src/IK/tests/HumanIKTest.cpp

```diff
@@ -9,6 +9,14 @@
 #include <BipedalLocomotion/ParametersHandler/StdImplementation.h>
 #include <BipedalLocomotion/ParametersHandler/TomlImplementation.h>
 #include <ConfigFolderPath.h>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <vector>
+#include <toml++/toml.h>
```

Comment:
```suggestion
```

It seems that these includes were used in previous iterations of this PR, and they are not needed anymore?